core/modules/system/system.module

```diff
@@ -3335,7 +3335,7 @@ function system_header_block($settings) {
 function system_breadcrumb_block($settings = array()) {
   $crumbs = backdrop_get_breadcrumb();
   if (isset($settings['current']) && $settings['current'] == TRUE) {
-    $crumbs[] = '<span>' . backdrop_get_title() . '</span>';
+    $crumbs[] = '<span aria-current="' . t('page') . "'>' . backdrop_get_title() . '</span>';
```

Comment:
This "page" shouldn't be translated, see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current for some details for aria-current and the possible values.

Reply:
@indigoxela I have removed translation for the page. Can you please check the changes?